src/modules/storage/LedgerStorage.ts

```diff
@@ -299,8 +299,16 @@ export class LedgerStorage extends Storages {
             last_updated_at   INTEGER,
             PRIMARY KEY (address(64))
         );
-        
 
+        CREATE TABLE IF NOT EXISTS account_history(
+            address          TEXT,
+            time_stamp       INTEGER NOT NULL,
+            balance          BIGINT(20) UNSIGNED NOT NULL,
+            granularity      TEXT NOT NULL,
+            
+            PRIMARY KEY (address(64), time_stamp, granularity(64))
+        );
+        
```

Comment:
https://github.com/bosagora/stoa/blob/v0.x.x/docs/Database_Schema.md
Please add the document.